components/protocol-wasm/world.wit

```diff
@@ -0,0 +1,67 @@
+/// An example world for the component to target.
+default world component {
+  export protocol: interface {
+    record proto-envelope-body {
+      content-bytes: list<u8>,
+      key-id: string,
+      signature: string
+    }
+    enum hash-algorithm {
+      sha256
+    }
+    record dyn-hash {
+      algo: hash-algorithm,
+      bytes: option<list<u8>>
+    }
+    variant record-id {
+      dyn-hash(dyn-hash)
+    }
+    record head {
+      digest: record-id,
+      timestamp: option<string>
+    }
+    enum perm {
```

Comment:
permission is not abbreviated throughout rest of the repo

Reply:
Stopped abbreviating